oper8/x/utils/deps_annotation.py

```diff
@@ -184,7 +191,7 @@ def get_deps_annotation(
                     kind,
                     name,
                 )
-                dep_dict = {}
+                continue
```

Comment:
So this removes the annotation instead of applying an empty one?

Reply:
Not quite. It avoids adding to the `sha1` object, so the result is no change from the base `hashlib.sha1().hexdigest()` if there are no found deps.